containers/vpn/ProtonVPNResourcesSection/ProtonVPNResourcesSection.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SubTitle, Row, Label, Field, Href } from 'react-components';
+import { c } from 'ttag';
+
+const ProtonVPNResourcesSection = () => {
+    return (
+        <>
+            <SubTitle>{c('Title').t`ProtonVPN Resources`}</SubTitle>
```

Comment:
Don't over capitalize please.
```suggestion
            <SubTitle>{c('Title').t`ProtonVPN resources`}</SubTitle>
```